### PR TITLE
Add top hits aggregation and order for terms aggregation

### PR DESCRIPTION
**Pull request by claymccoy**

### CODE REVIEW

File: elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/AggregationDsl.scala [hunk 1]
```diff
@@ -42,6 +42,7 @@ trait AggregationDsl extends DslCommons with QueryDsl {
                               size: Option[Int], shardSize: Option[Int],
                               hint: Option[String] = None,
                               name: Option[String] = None,
+                              order: Option[SortOrder] = None,
```
Comment (rcoh) [inline elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/AggregationDsl.scala]: can you make order last so we retain backwards compatibility?

**Review by rcoh — CHANGES_REQUESTED** [review]
Just one small change, thanks for your contribution!

File: elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/AggregationDsl.scala (hunk 1 above)
Comment (claymccoy) [reply elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/AggregationDsl.scala]: Yeah, I almost did that, but it seemed fitting to have nested aggs at the end. I'll be happy to change it though.

**Comment by claymccoy** [comment]
Changed parameter order and rebased with master to get #91 and test locally.

**Pull request merged**